lib/rubocop/cop/style/ternary_parentheses.rb

```diff
@@ -34,57 +34,103 @@ module Style
       #   foo = (bar?) ? a : b
       #   foo = (bar.baz) ? a : b
       #   foo = (bar && baz) ? a : b
+      #
+      # @example
+      #
+      #   EnforcedStyle: require_parentheses_when_complex
+      #
+      #   @bad
+      #   foo = (bar?) ? a : b
+      #   foo = (bar.baz?) ? a : b
+      #   foo = bar && baz ? a : b
+      #
+      #   @good
+      #   foo = bar? ? a : b
+      #   foo = bar.baz ? a : b
+      #   foo = (bar && baz) ? a : b
       class TernaryParentheses < Cop
         include IfNode
         include SafeAssignment
         include ConfigurableEnforcedStyle
 
         MSG = '%s parentheses for ternary conditions.'.freeze
+        MSG_COMPLEX = 'Use parentheses %sfor ternary conditions with' \
+          ' complex expressions.'.freeze
```

Comment:
%sfor seems wrong to me. 

Ternary expressions with complex conditions. Guess you swapped the words here.